packages/svelte/src/internal/client/dom/elements/bindings/media.js

```diff
@@ -34,22 +35,21 @@ export function bind_current_time(media, get_value, update) {
 			raf_id = requestAnimationFrame(callback);
```

Comment:
I wonder if a rAF loop is right here because media should play when off tab, but the rAF will not run at all.

Reply:
should be fine because once you switch back to the tab it continues to loop, and while you're off tab whatever the binding update causes wouldn't be of interest anyway